src/plugins/auto/multi_schedule.cpp

```diff
@@ -13,7 +13,7 @@ namespace MultiDevicePlugin {
 thread_local WorkerInferRequest* MultiSchedule::_thisWorkerInferRequest = nullptr;
 // TODO: revert to the plain variable (see header file), when we moved to the next CentOS 8.x in our support matrix
 thread_local const char* MultiSchedule::_thisPreferredDeviceName = "";
-
+thread_local SoExecNetwork MultiSchedule::_passthroughExeNet = {};
```

Comment:
Is it possible that this MultiSchedule  instance will be created twice in the same thread? If yes, thread_local will be reused , is it expected behavior?

Reply:
yes, good point, this is not an elegant way to fix this issue and I do see potential failures. Generally I think it's not the correct behavior for virtual plugin(auto/hetero/batch) to pass hw device blobs directly to app, because the _so dependency is not correct in this case, and may lead to potential issues.